numpy/core/tests/test_mem_policy.py

```diff
@@ -35,18 +35,24 @@ def get_module(tmp_path):
     prologue = '''
         #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
         #include <numpy/arrayobject.h>
+        typedef struct {
```

Comment:
Shouldn't this have the ctx field as well?

Reply:
This is actually an example `ctx` struct itself.

#L109 you will find a `ctx` of this type. #L115 you will find the `PyDataMem_Handler` that makes use of that `ctx`. 

Tell me if you have any suggestions for another (simpler ?) testing of the `ctx` functionality.